tee_attestation/src/lib.rs

```diff
@@ -13,25 +25,70 @@ pub enum TeeType {
 }
 
 pub enum TeeQuote {
-    Sev(Box<SEVQuote>),
-    Sgx(Box<SGXQuote>),
-    Tdx(Box<TDXQuote>),
+    Sev(Box<SevQuote>),
+    Sgx(Box<SgxQuote>),
+    Tdx(Box<TdxQuote>),
 }
 
-#[derive(Debug)]
-pub struct SgxMeasurement {
-    pub public_signer_key_pem: String,
-    pub mr_enclave: [u8; 32],
+#[derive(Default, Debug, Clone, Deserialize, Serialize, PartialEq, Copy)]
+pub struct TeePolicy {
+    pub sgx: Option<SgxQuoteVerificationPolicy>,
+    pub sev: Option<SevQuoteVerificationPolicy>,
+    pub tdx: Option<TdxQuoteVerificationPolicy>,
 }
```

Comment:
looking at impl below, it seems that it's more appropriate to use an `Enum` instead (just like `TeeQuote`), as 2 fields are set to `None` when the third one is filled in. What do you think ?

Reply:
Good point of view. Two thoughts:
- All fields can be None. But i guess we can add a forth attribute "none"
- On the user point of view, it can set severals policies. One is active per environement but by doing that its application is full portable

Reply:
I will think deeper about that considering how we use it in ralts and cosmian vm tools